functions.php

```diff
@@ -1,55 +1,78 @@
 <?php
 
-function getAllFilmsWithoutRoles(string $db_name):  array
-{
+/**
+ * Creates a PDO connection object to the DB Name thats passed in
+ */
+function getDBConnection(string $db_name): object {
```

Comment:
Because you're using the PDO class you want to type hint to PDO, I think object still sortof works, but it could then be any data of type object.